### PR TITLE
add create-checkpoint cmd

### PR DESCRIPTION
**Pull request by zhangchiqing**
## Goal of this PR

- Add create-checkpoint cmd to create a checkpoint
- Add payloads cmd to read register by owner and key at a given height
- Add docker build commands for cmd

### CODE REVIEW

**Review by pattyshack — APPROVED** [review]

File: cmd/payloads/main.go [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+	}
+
+	if flagHeight == 0 {
+		log.Error().Msgf("--height flag is 0")
+		return failure
+	}
+
+	err = GetPayload(flagIndex, flagHeight, flagOwner, flagKey, log)
+	if err != nil {
+		log.Error().Err(err).Msg("can not create checkpoint")
```
Comment (peterargue) [inline cmd/payloads/main.go:77]:
```suggestion
		log.Error().Err(err).Msg("can not get payload")
```

**Review by peterargue — APPROVED** [review]
looks good. Can you add comment blocks to each of the command files describing what they do?

**Comment by zhangchiqing** [comment]
Tested works:

## Create checkpoint
```
root@mainnet23-leo:/var/flow# ./create-checkpoint --checkpoint /var/flow/checkpoint
{"level":"info","index":"/var/flow/data/pebble/index2","level":"info","checkpoint_dir":"/var/flow/checkpoint","time":"2023-06-29T14:38:55Z","message":"flags loaded"}
{"level":"info","time":"2023-06-29T14:38:55Z","message":"successfully created checkpoint at dir: /var/flow/checkpoint"}
```

## Get payload from storage
```
root@mainnet23-leo:/var/flow# ./payloads --key "uuid" --height 55630691
{"level":"info","index":"/var/flow/data/pebble/index2","level":"info","height":55630691,"owner":"","key":"uuid","time":"2023-06-29T14:50:37Z","message":"flags loaded"}
{"level":"info","time":"2023-06-29T14:50:37Z","message":"successfully get register value at height 55630691 for reg id: /#75756964: 0000000044ddbd63 (len: 8)"}
```

## Get the same payload from checkpoint
```
{"level":"info","index":"/var/flow/checkpoint","level":"info","height":55630691,"owner":"","key":"uuid","time":"2023-06-29T14:50:39Z","message":"flags loaded"}
{"level":"info","time":"2023-06-29T14:50:39Z","message":"successfully get register value at height 55630691 for reg id: /#75756964: 0000000044ddbd63 (len: 8)"}
```

**Comment by zhangchiqing** [comment]
bors merge

**Pull request merged**